datasets/za/fic_sanctions/crawler.py

```diff
@@ -94,10 +94,9 @@ def crawl_row(context: Context, data: Dict[str, str]):
         for alias in h.multi_split(aliases, [", ", "Good", "Low"]):
             if all(c in {"?", " "} for c in alias):
                 continue
-            if "?" in alias:
-                context.log.warning("Alias contains '?', skipping", alias=alias)
-                continue
             entity.add("alias", alias)
+            if "?" in entity.get("alias"):
+                context.log.warning("Alias contains '?'", alias=alias)
```

Comment:
this is saying "if any entire alias in the alias list is `?`"
I think what you want is this:
```suggestion
            if any(["?" in a for a in entity.get("alias")]):
                context.log.warning("Alias contains '?'", alias=alias)
```